ci: don't run ivy and aio_monitoring CI jobs on the patch branch

IgorMinar opened the pull request:
We don't need these jobs to run on the patch branch - it would only drain
our VM queue.

alexeagle reviewed (approved):
possible alternative: keep this file the same between master and patch, but bail out:

```
      - run:
          name: Skip this job for Pull Requests and Fork builds
          # Note, `|| true` on the end makes this step always exit 0
          command: '[[
              -v CIRCLE_PR_NUMBER
              || "$CIRCLE_PROJECT_USERNAME" != "angular"
              || "$CIRCLE_PROJECT_REPONAME" != "angular"
          ]] && circleci step halt || true'
```

IgorMinar committed:
ci: disable ivy CI jobs on the patch branch

we don't need these jobs to run on the patch branch - this will free up VMs for other jobs
